src/wrapper/jnienv.rs

```diff
@@ -692,14 +689,16 @@ impl<'a> JNIEnv<'a> {
                     jni_args
                 )
                 .into(),
-                Primitive::Void => jni_non_void_call!(
-                    self.internal,
-                    CallStaticVoidMethodA,
-                    class,
-                    method_id,
-                    jni_args
-                )
-                .into(),
+                Primitive::Void => {
+                    jni_non_void_call!(
```

Comment:
**non_void**_call to call a void method?
That looks suspicious :thought_balloon:

Reply:
Ooops... 🙄